capa_dominio/AreaTrabajo.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace capa_dominio
+{
+    public class AreaTrabajo
+    {
+        public int AreaId { get; private set; }
+        public string AreaNombre { get; private set; }
+        public string Descripcion { get; private set; }
+
+        public AreaTrabajo(int id, string nombre, string descripcion)
```

Comment:
Invalido, no es validacion correcta , no es regla propia de la clase , si en la tabla dice no null , es decir el nombre de la area siempre va a llegar nunca sera null esa validacion nunca se lanzara por lo tanto inecesario , , creo que aqui no hay metodo a menos que venga el estado , si es asi el metodo tiene que ser de tipo bool y decir ,  el area esta activo?? y se acabo. la estructura es elAreaEsActivo() pero no es el caso 
```suggestion
        public AreaTrabajo(int id, string nombre, string descripcion)
        {
            AreaId = id;
            AreaNombre = nombre;
            Descripcion = descripcion;

            if (string.IsNullOrWhiteSpace(nombre))
                throw new ArgumentException("El nombre del área es obligatorio.");
        }
    }
}
```